test/integration/appTemplates.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { ToolpadHome } from '../models/ToolpadHome';
+import { ToolpadRuntime } from '../models/ToolpadRuntime';
+import { test, expect } from '../playwright/test';
+
+test('can use statistics app template', async ({ page }) => {
+  const homeModel = new ToolpadHome(page);
+  await homeModel.goto();
+  const app = await homeModel.createApplication({ appTemplateId: 'stats' });
+
+  page.waitForNavigation();
+
+  const runtimeModel = new ToolpadRuntime(page);
+  await runtimeModel.gotoPage(app.id, 'stats');
+
+  await page.locator('h3:has-text("Statistics")').waitFor({ state: 'visible' });
```

Comment:
when we use `tag:has-text(*)` instead of `text=*` is it because we have multiple occurrences of the same text and we need more specific selector?

Reply:
Just trying to be more specific and trying to avoid issues with multiple occurrences in general, not just now but in the future.

Reply:
Should you then update other locators that currently use only `text=*`?

Reply:
Those are just divs, so it feels less specific...
I'll give them another look though.